NOTICE.txt

```diff
@@ -1071,6 +1071,46 @@ Most icons in the Spyder 3 theme; used via Spyder's own QtAwesome package.
 See spyder/utils/icon_manager.py for a complete list.
 
 
+-------------------------------------------------------------------------------
+
+
+Material Design (via QtAwesome)
+--------------------------------
+
+
+Copyright 2014 Austin Andrews
+
+
+Author: Austin Andrews/MaterialDesign
+Site: http://materialdesignicons.com/
+Source: https://github.com/Templarian/MaterialDesign
+License: SIL Open Font License 1.1 | https://scripts.sil.org/OFL
+
+No modifications made.
+
+
+Copyright (c) 2014, Austin Andrews (http://materialdesignicons.com/),
+with Reserved Font Name Material Design Icons.
+Copyright (c) 2014, Google (http://www.google.com/design/)
+uses the license at https://github.com/google/material-design-icons/blob/master/LICENSE
```

Comment:
This block belongs at the top, replacing the ``Copyright 2014 Austin Andrews`` line.